pkg/logs/sds/scanner.go

```diff
@@ -162,34 +186,42 @@ func (s *Scanner) reconfigureRules(rawConfig []byte) error {
 			s.Scanner = nil
 			s.rawConfig = rawConfig
 			s.configuredRules = nil
-			return nil
+			tlmSDSReconfigSuccess.Inc("shutdown")
 		}
 		return nil
 	}
 
 	// prepare the scanner rules
 	var sdsRules []sds.Rule
+	var malformedRulesCount int
 	for _, userRule := range config.Rules {
 		// read the rule in the standard rules
 		standardRule, found := s.standardRules[userRule.Definition.StandardRuleID]
 		if !found {
 			log.Warnf("Referencing an unknown standard rule, id: %v", userRule.Definition.StandardRuleID)
+			tlmSDSUnknownStdRule.Inc()
 			continue
 		}
 
 		if rule, err := interpretRCRule(userRule, standardRule); err != nil {
 			// we warn that we can't interpret this rule, but we continue in order
 			// to properly continue processing with the rest of the rules.
+			malformedRulesCount += 1
```

Comment:
Not that there is much practical difference, but is there any reason to count this and `tlmSDSUnknownStdRule` in two distinct ways?